hack/dockerfile/install-binaries.sh

```diff
@@ -47,11 +47,15 @@ install_proxy() {
 }
 
 install_dockercli() {
-	echo "Install docker/cli version $DOCKERCLI_COMMIT"
-	git clone "$DOCKERCLI_REPO" "$GOPATH/src/github.com/docker/cli"
-	cd "$GOPATH/src/github.com/docker/cli"
-	git checkout -q "$DOCKERCLI_COMMIT"
-	go build -o /usr/local/bin/docker github.com/docker/cli/cmd/docker
+	DOCKERCLI_CHANNEL=${DOCKERCLI_CHANNEL:-edge}
+	DOCKERCLI_VERSION=${DOCKERCLI_VERSION:-17.06.0-ce}
+	echo "Install docker/cli version $DOCKERCLI_VERSION from $DOCKERCLI_CHANNEL"
+
+	url=https://download.docker.com/linux/static
+	curl -Ls $url/$DOCKERCLI_CHANNEL/x86_64/docker-$DOCKERCLI_VERSION.tgz | \
```

Comment:
p/z failures are real. While s390x and armhf are in that channel and can be referenced here, ppc64le, arm64, and to a lesser extent i386 aren't so they would still have to be built from src.

If the cli binary corresponds directly to the upstream repo, it might make sense to have version tags in the cli repo.